challenges/fizz_buzz_tree/test_fizzbuzz.py

```diff
@@ -0,0 +1,32 @@
+from .bst import BST
+from .fizzbuzztree import fizzbuzztree as fb
+import pytest
+
+
+@pytest.fixture
+def new_bst():
+    return BST()
+
+
+@pytest.fixture
+def filled_bst():
+    return BST([4, 3, 2, 1, 8, 6, 12, 9])
+
+
+def test_post_order_(new_bst):
+    """empty bst"""
+    assert fb(new_bst) is False
+
+
+def test_data_bst_fizzbuzztree_in_order_traverse(filled_bst):
+    filled_bst = fb(filled_bst)
+    lst = []
+    filled_bst.in_order(lst.append)
+    assert lst == [1, 2, 'fizz', 4, 'fizz', 8,  'fizz', 'fizz']
+
+
+def test_data_bst_fizzbuzztree_pre_order_traverse(filled_bst):
+    filled_bst = fb(filled_bst)
+    lst = []
+    filled_bst.pre_order(lst.append)
+    assert lst == [4, 'fizz', 2, 1, 8, 'fizz', 'fizz', 'fizz']
```

Comment:
This is the same as the previous test. These tests are supposed to be checking the functionality of your `fizzbuzztree` function. Passing the same tree to `fizzbuzztree` does not check different aspects of your function, even if you traverse the resulting tree in pre-order instead of in-order.

You might want to test a tree that has values that would generate `'buzz'` and/or `'fizzbuzz`' since this tree only generates `'buzz'`s.